docs/src/pages/premium-themes/onepirate/modules/components/Typography.js

```diff
@@ -102,6 +102,8 @@ Typography.propTypes = {
     'body2',
     'button',
     'caption',
+    'dark',
```

Comment:
premium theme?

Reply:
so change it to 'premium' instead of 'dark' for when contrast values are used?

Reply:
sorry for the confusion on this comment! 
looks like that file gets automatically altered upon running yarn docs:typescript:formatted

i think it's a docs file that lays out all possible variants off of typography and since we are adding some there, it gets modified when running the docs formatted command